web/src/controllers/Scripts.ts

```diff
@@ -118,7 +118,7 @@ export class Installers {
 
     response.set("X-Kurl-Hash", installer.hash());
     response.status(200);
-    return this.templates.renderInstallScript(installer, kurlVersion);
+    return this.templates.renderInstallScript(installer, installer.spec.kurl?.installerVersion);
```

Comment:
wont this make kurlversion in the url no longer work?

Reply:
no, because of the code change to have the version in the URL inserted into the installer spec

Reply:
so what takes precedence? what should take precedence?

Reply:
This handler is not used to get installers for custom specs, this just gets the latest installer or a specific version that is specified in the URL. With this change the kurlVersion from the URL will be added into the spec (line 117), at the end of the day these two would be the same: 
`return this.templates.renderInstallScript(installer, kurlVersion);`
`return this.templates.renderInstallScript(installer, installer.spec.kurl?.installerVersion);`